packages/legend-query/src/components/QueryBuilderValueSpecificationEditor.tsx

```diff
@@ -534,75 +599,601 @@ export const LatestDatePrimitiveInstanceValueEditor: React.FC = () => (
   </div>
 );
 
+export const AbsoluteDateValueSpecificationEditor: React.FC<{
```

Comment:
Could you do me a favor and remove `export` from everything in this file except for `QueryBuilderValueSpecificationEditor`?